spec/fabricators/user_fabricator.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Fabricator(:user) do
+  lastname 'DENT'
```

Comment:
We can try to use https://github.com/ffaker/ffaker gem as well if required.

Reply:
I'll try it after lunch! Cool!! :)

Reply:
Solved! Wow, very easy to use! :)